### PR TITLE
extends #1028 - Add a test that checks, that no health checks will be st...

### PR DESCRIPTION
**Pull request by drexin**
...arted for staging tasks

### CODE REVIEW

**Comment by drexin** [comment]
Merge only after #1028 has been merged.

**Comment by sttts** [comment]
Thanks!

**Comment by ConnorDoyle** [comment]
Build failure:

```
[info] HealthCheckActorTest:
[INFO] [01/16/2015 16:04:50.946] [pool-1-thread-1-ScalaTest-running-HealthCheckActorTest] [akka://system/user/$$A] Starting health check actor for app [test] and healthCheck [HealthCheck(Some(/),HTTP,0,None,15 seconds,10 seconds,20 seconds,3)]
[info] - should not dispatch health checks for staging tasks *** FAILED ***
[info]   true was not false (HealthCheckActorTest.scala:45)
```

**Comment by sttts** [comment]
Will rebase this one on top of my patch. Otherwise, the test will fail obviously.

**Comment by drexin** [comment]
Yep, this is the regression I'm testing ;-)

**Comment by drexin** [comment]
Rebase with https://github.com/mesosphere/marathon/pull/1028

**Comment by sttts** [comment]
Great, thanks.

File: src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala [hunk 1]
```diff
@@ -51,10 +52,7 @@ class HealthCheckActor(
     )
   }
 
-  // self-sent every healthCheck.intervalSeconds
```
Comment (ConnorDoyle) [inline src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala]: OK to move `Tick` to the companion object, but can we carry this comment there as well?

**Comment by ConnorDoyle** [comment]
Tiny nit regarding a lost comment, feel free to :ship: it once that's fixed.

**Pull request merged**